guides/assets/stylesheets/main.css

```diff
@@ -280,6 +280,10 @@ body {
   #feature .wrapper { padding-right: 0; }
 }
 
+@media screen and (min-width: 800px) and (max-width: 960px) {
+  #container .wrapper { padding-right: 21em; }
+}
+
 /* Links
```

Comment:
Here, if you move your media query above the one on line 279 you can drop the `and (min-width: 800)` since it would be implied due to cascading. You'd have to reset the `#container .wrapper {padding-right: 0}` inside the `max-width:800` media query but  using the cascade instead of declaring the min-width seems to be more in line with how media queries are being used in the rest of the file.